R/project.R

```diff
@@ -185,19 +185,33 @@ Project <-
       #' @description Export the entire project to a JSON file
       #' @param filePath The file path where the JSON will be saved
       #' @return NULL
-      exportToJSON = function() {
+      exportToJSON = function(interactiveInput = TRUE, dirPath = NULL, fileName = NULL) {
```

Comment:
Also provide an `export()` **function** defined like this:

`export <- function(project, json = TRUE, xlsx = TRUE){}`

This function would call internally `project$exportToJSON` and in the future `project$exportToExcel` once we have it.